docs/Demo/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from '@deque/cauldron-react/';
+
+const Footer = props => {
+  const year = new Date().getFullYear();
+
+  return (
+    <footer className="Footer">
+      <ul className="Footer__links">
+        <li>
+          <Link href="https://www.deque.com/terms-of-use/">Terms of Use</Link>
+        </li>
```

Comment:
I'm wondering if we should use external icons or the words "external" here, but I'm not sure how strict we need to be on this.

This resources says, "A link to an external site MAY indicate that it leads to an external site.": https://dequeuniversity.com/checklists/web/links

Reply:
🤔 I think the main thing to warn users about links is if it opens in a new tab, and these open in the same tab?

Reply:
[This example at the bottom](https://dequeuniversity.com/class/semantic-structure/links/external-windows-files#external-site) doesn't have it opening in a new tab but indicates that it's external.

<img width="824" alt="Screen Shot 2022-09-15 at 12 59 35 PM" src="https://user-images.githubusercontent.com/3081483/190487266-2ca433c0-2b1e-44e9-b896-b09925e02507.png">

Reply:
And yeah that same URL above has "SHOULD" for "A link that opens in a new window or tab SHOULD indicate that it opens in a new window or tab."